### PR TITLE
Update app-sdk-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that If the App Protection Policy is related to Managed Devices, it is necessary to create an App Configuration Profile of the application that has Intune integrated.

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3776

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/developer/app-sdk-ios.md [hunk 1]
```diff
@@ -222,6 +223,10 @@ Alternatively, apps can override these Azure AD settings at runtime. To do this,
 
 4. Ensure the steps to give your iOS app permissions to the app protection policy (APP) service are followed. Use the instructions in the [getting started with the Intune SDK guide](app-sdk-get-started.md#next-steps-after-integration) under "[Give your app access to the Intune app protection service (optional)](app-sdk-get-started.md#give-your-app-access-to-the-intune-app-protection-service-optional)".  
 
+> [!NOTE]
+> If the App Protection Policy is related to Managed Devices, it is necessary to create an App Configuration Profile of the application that has Intune integrated.
+
+
 > [!NOTE]
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/app-sdk-ios.md:230]:
```suggestion
> If the app protection policy is related to managed devices, creating an app configuration profile of the application that has Intune integrated is also necessary.
>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba remove warnings.

**Comment by msbemba** [comment]
@yogkumgit The warnings are for the Header in the document which I think only the Author can add.

Thanks
Sri

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**